substrate/client/network/src/request_responses.rs

```diff
@@ -808,7 +919,8 @@ impl NetworkBehaviour for RequestResponsesBehaviour {
 								}) => {
 									// Try using the fallback request if the protocol was not
 									// supported.
-									if let OutboundFailure::UnsupportedProtocols = error {
+									if let OutboundFailure::UnsupportedProtocols = error {	
+										error_bis_out = CustomOutboundFailure::UnsupportedProtocols;									
```

Comment:
A global variable should not be used. The conversion should be performed in place.